c++/src/kj/async.c++

```diff
@@ -216,6 +240,13 @@ public:
   Maybe<Own<Task>> next;
   Maybe<Own<Task>>* prev = nullptr;
 
+  kj::String trace() {
```

Comment:
I couldn't find where this, and PromiseNode::trace(), are called. Are they used?

Reply:
`TaskSet::Task::trace()` is called by `TaskSet::trace()`. `PromiseNode::trace()` is called by `Promise::trace()`. `TaskSet::trace()` and `Promise::trace()` are public interfaces.